app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -2,7 +2,6 @@ class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::
   needs :@record
 
   def visible?
```

Comment:
In that case, you don't even need to use `HostFeatureButton` class, but you can use `GenericFeatureButton`, just make sure that it is used on some other places, if it's not, it can be completely removed.